docs/introduction/upgrading-to-stencil-four.md

```diff
@@ -0,0 +1,46 @@
+---
+title: Upgrading to Stencil v4.0.0
+description: Upgrading to Stencil v4.0.0
+url: /docs/upgrading-to-stencil-4
+---
+
+# Upgrading to Stencil v4.0.0
+
+:::caution 
+Stencil 4.0.0 is pre-release software.
+These instructions are for users looking to try an early version of the software
+:::
+
+## Getting Started
+
+It's recommended that your projects start their upgrade from Stencil v3.
+Projects using Stencil v0 or Stencil v1 should upgrade to Stencil v2 before proceeding with upgrading to Stencil v3.
```

Comment:
What I had been going for was something to the effect of "Don't upgrade from v1 to v4 directly. Instead, go one major version at a time." Maybe something like this?

> It is suggested that projects using a version of Stencil prior to v3 should upgrade one major version at a time, rather than upgrading from v0 directly to v3.

Reply:
Oh gotcha. Makes sense

Reply:
yeah I think that sounds good to replace this and the following sentence